examples/runners/ash/src/main.rs

```diff
@@ -236,7 +236,7 @@ pub fn main() {
                 WindowEvent::CloseRequested => event_loop_window_target.exit(),
                 _ => {}
             },
-            _ => event_loop_window_target.set_control_flow(ControlFlow::Wait),
+            _ => event_loop_window_target.set_control_flow(ControlFlow::Poll),
```

Comment:
Ah, is this the change that makes it animate continually instead of only while activating the mouse/keyboard?